dbms/src/Storages/FormatVersion.h

```diff
@@ -118,7 +118,7 @@ inline static const StorageFormatVersion STORAGE_FORMAT_V4 = StorageFormatVersio
     .identifier = 4,
 };
 
-inline StorageFormatVersion STORAGE_FORMAT_CURRENT = STORAGE_FORMAT_V4;
+inline StorageFormatVersion STORAGE_FORMAT_CURRENT = STORAGE_FORMAT_V3;
```

Comment:
```suggestion
// Note that we don't enable V4 by default because https://github.com/pingcap/tiflash/issues/5076
// is not yet figure out.
inline StorageFormatVersion STORAGE_FORMAT_CURRENT = STORAGE_FORMAT_V3;
```